### PR TITLE
feat(Omnichannel): Attachments in Email Transcripts

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes (including videos or screenshots)

File attachments were not being handled in email transcripts, so implemented file attachments in email transcripts and invalid attachment message for attachments other than images.

image

## Further comments

[CORE-554](https://rocketchat.atlassian.net/browse/CORE-554)

[CORE-554]: https://rocketchat.atlassian.net/browse/CORE-554?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/weak-taxis-design.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+Fixed handling of attachments in Omnichannel email transcripts. Earlier attachments were being skipped and were being shown as empty space, now it should render the image attachments and should show relevant error message for unsupported attachments.
```
Comment (MarcosSpessatto) [inline .changeset/weak-taxis-design.md:5]: I believe we didn't fix a problem here, we added the support for it instead, right?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
Is there any way of adding some tests here?

**Review by tiagoevanp — COMMENTED** [review]
The description of PR is confusing, can you add a similar message from the changeset and show in the images who is before/after?

File: .changeset/weak-taxis-design.md (hunk 1 above)
Comment (tiagoevanp) [reply .changeset/weak-taxis-design.md:5]: Same, PR title says `feat`...

File: apps/meteor/app/livechat/server/lib/LivechatTyped.ts [hunk 2]
```diff
@@ -623,21 +626,73 @@ class LivechatClass {
...
+
+					const uploadedFile = await Uploads.findOneById(file._id);
+
+					if (!uploadedFile) {
+						filesHTML += `<div>${file.name}${InvalidFileMessage}</div>`;
+						continue;
+					}
+
+					const uploadedFileBuffer = await FileUpload.getBuffer(uploadedFile);
+					filesHTML += `<div>${file.name}<img src="data:${attachment.image_type};base64,${uploadedFileBuffer.toString(
```
[the remark this answers is not in the document]
Comment (yash-rajpal) [reply apps/meteor/app/livechat/server/lib/LivechatTyped.ts]: Done!

**Review by KevLehman — DISMISSED** [review]

**Review by KevLehman — APPROVED** [review]

**Pull request merged**